src/components/constants/display-sections.js

```diff
@@ -1 +1,2 @@
 export const CHALLENGES_SCREEN = 'Challenges';
+export const dummyProfile = 'dummyProfile.png';
```

Comment:
Please make **dummyProfile** as upper case and words must be separated by underscores

Reply:
The idea here is that it's a convention (not a rule) that we keep the those variables in all uppercase -- whose value is known to us before the program starts executing, is not calculated during compile time and does not change throughout the execution. It's easier for a developer to understand the code while going through it "okay, this is a constant known to us beforehand and is not calculated during runtime, its value must be declared somewhere, let me go and check"

Hope that makes sense @abhinav1590 

Also @Pavangbhat good that you pointed it out but please mention reasons also, as to why you think something should be done. You don't have to type huge paragraphs but a one-liner or a blog/article/github or stackoverflow thread would also do the job. The point is that everyone should understand why they're doing what they're doing, not just because some reviewer told them to do so.

Reply:
Thank you @swarajpure , Yes I should have explained it. Next time for sure will keep this in mind